### PR TITLE
Unstale on comment

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for ff5cf3f26b7dcd5e1483c36238ec9cdf2c569cb9:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1471-a4498905004a7dcbfcb85cae558d3b329aa8d99a-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1471-a4498905004a7dcbfcb85cae558d3b329aa8d99a-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1471-a4498905004a7dcbfcb85cae558d3b329aa8d99a-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1471-a4498905004a7dcbfcb85cae558d3b329aa8d99a-x86.apk)

**Comment by m-kuhn** [comment]
@suricactus does this look safe to you?

File: .github/workflows/unstale.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+jobs:
+  remove_labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-ecosystem/action-remove-labels@v1
+        if: ${{ github.event.comment.user.url != 'https://github.com/apps/github-actions' }}
+        with:
+          github_token: ${{ secrets.github_token }}
```
Comment (suricactus) [inline .github/workflows/unstale.yml:13]: I guess it is case insensitive, but can we use secrets.GITHUB_TOKEN instead, as we do in the rest of the workflows?

**Review by suricactus — APPROVED** [review]

File: .github/workflows/unstale.yml (hunk 1 above)
Comment (m-kuhn) [reply .github/workflows/unstale.yml:13]: Good point, it was copy pasted

```suggestion
          github_token: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 6232785ec3bd2f6368c6eed6df9b7bf05be3ecb3:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1471-9a03c7c7c47513d3a97eab6f08bd60330ff6aaa7-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1471-9a03c7c7c47513d3a97eab6f08bd60330ff6aaa7-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1471-9a03c7c7c47513d3a97eab6f08bd60330ff6aaa7-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1471-9a03c7c7c47513d3a97eab6f08bd60330ff6aaa7-x86.apk)

**Pull request merged**